cornflow-server/cornflow/schemas/execution.py

```diff
@@ -114,6 +114,10 @@ def get_indicators(self, obj):
     indicators = fields.Method("get_indicators")
 
 
+class ExecutionDetailsWithIndicatorsAndLogResponse(ExecutionDetailsEndpointWithIndicatorsResponse):
+    basic_log_json = fields.Nested(BasicLogSchema, attribute="log_json")
```

Comment:
I would not call this field `basic_log_json`. I think with a name as `log` it should be enough, in this case it only bring some information instead of the full log, but it should be enough.